src/analysis/inner_product_space/pi_L2.lean

```diff
@@ -30,8 +31,19 @@ This is recorded in this file as an inner product space instance on `pi_Lp 2`.
 - `linear_isometry_equiv.of_inner_product_space`: provides an arbitrary isometry to Euclidean space
   from a given finite-dimensional inner product space, induced by choosing an arbitrary basis.
 
+- `std_orthonormal_basis`: the canonical `orthonormal_basis` on `euclidean_space 𝕜 ι`
+
 - `complex.isometry_euclidean`: standard isometry from `ℂ` to `euclidean_space ℝ (fin 2)`
 
+The last section covers orthonormal bases, etc. The lemma
+`maximal_orthonormal_iff_orthogonal_complement_eq_bot` states that an orthonormal set in an inner
+product space is maximal, if and only the orthogonal complement of its span is trivial.
+Various consequences are stated for finite-dimensional `E`, including that a maximal orthonormal
+set is a basis (`maximal_orthonormal_iff_basis_of_finite_dimensional`); these consequences require
+the theory on the orthogonal complement developed earlier in this file.  For consequences in
+infinite dimension (Hilbert bases, etc.), see the file `analysis.inner_product_space.l2_space`.
```

Comment:
This docstring needs a complete rewrite after all your good work!

Reply:
Rewritten! Let me know what you think